VDA-Name-Registry/contracts/VeridaDataVerificationLib.sol

```diff
@@ -16,8 +16,13 @@ library VeridaDataVerificationLib {
     bytes memory _unsignedData,
     bytes calldata _signature,
     address _signerAddress
-  ) internal pure returns (bool) {
+  ) internal pure returns (bool result) {
```

Comment:
Shouldn't this be deleted and the re-usable library used instead?

Reply:
No, we use separate VeridaDataVerficationLib for `DID-Registry` and `NameRegistry` contracts.